r-package/grf/R/plot.R

```diff
@@ -88,6 +88,20 @@ export_graphviz <- function(tree) {
 #' @param ... Additional arguments (currently ignored).
 #'
 #' @method plot grf_tree
+#' @examples
```

Comment:
Great! there was a \dontrun missing so the final version could look like:

```
#' \dontrun{
#' Saving a tree plot programmatically requires the "DiagrammeRsvg" package:
#' # install.packages("DiagrammeRsvg")
#' n <- 500
#' p <- 10
#' X <- matrix(rnorm(n * p), n, p)
#' W <- rbinom(n, 1, 0.5)
#' Y <- pmax(X[, 1], 0) * W + X[, 2] + pmin(X[, 3], 0) + rnorm(n)
#' c.forest <- causal_forest(X, Y, W)
#' # Save the first tree in the forest as plot.svg
#' tree.plot = plot(get_tree(c.forest, 1))
#' cat(DiagrammeRsvg::export_svg(tree.plot), file = 'plot.svg')
#'}
```

Also, please remove the .Rapp.history committed file

Tip: if you have grf open with the grf.Rproj file you can build the documentation with CMD+Shift+D, then reload grf with CMD+Shift+L, the look at the generated documentation with ?plot.grf_tree